web/src/beta/features/Editor/DataSourceManager/DelimitedText/index.tsx

```diff
@@ -64,20 +79,34 @@ const DelimitedText: React.FC<DataProps> = ({ sceneId, onSubmit, onClose }) => {
           description="Select the type of data source you want to add.">
           <SourceTypeWrapper>
             <RadioGroup
-              options={[{ label: t("From URL"), keyValue: "url" }]}
+              options={DataSourceOptions}
               selectedValue={sourceType}
               onChange={setSourceType}
             />
           </SourceTypeWrapper>
         </InputGroup>
-        <InputGroup label="Resource URL" description="URL of the data source you want to add.">
-          <Input
-            type="text"
-            placeholder="Input Text"
-            value={value}
-            onChange={e => setValue(e.target.value)}
-          />
-        </InputGroup>
+
+        <SelectField
+          value={fileFormat}
+          options={["CSV"].map(v => ({ key: v, label: v }))}
```

Comment:
We will have more options in the future, right? Can we movfe this array up as a constant 🙏🏻